modules/global-classes/global-classes-rest-api.php

```diff
@@ -87,7 +92,7 @@ private function register_routes() {
 				'methods' => 'POST',
 				'callback' => fn( $request ) => $this->route_wrapper( fn() =>  $this->create( $request ) ),
 				'validate_callback' => function( \WP_REST_Request $request ) {
-					[ $is_valid ] = Style_Validator::make( Style_Schema::get() )
+					[ $is_valid ] = Style_Parser::make( Style_Schema::get() )
```

Comment:
We should also sanitize in the `create` flow

Reply:
Done (including tests)